README.md

```diff
@@ -292,11 +292,51 @@ extension that you created, and pass that into `StyleSheet.extend()`:
 ```js
 const mySelectorHandler = ...;
 
-const myExtension = { selectorHandler: mySelectorHandler };
+const myExtension = {selectorHandler: mySelectorHandler};
 
 StyleSheet.extend([myExtension]);
```

Comment:
Ah yes, that is correct. I'm not sure how to indicate that. `module.exports = `? `const {StyleSheet, css} = `?